index.html

```diff
@@ -208,6 +208,49 @@ <h2 class="skills"><span><img src="./Image/Rectangle-57.png" alt=""></span>Skill
 
             </div>
         </div>
+        <div class="contact-form">
+            <h1>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+            <div class="form">
+                <form action="https://formspree.io/f/xeqnpraa" method="POST">
+                    <label><input type="text" name="name" maxlength="30" required=""
+                            placeholder="Full Name"></label>
+                    <label><input type="email" name="email" required="" placeholder="Email Address"></label>
+                    <label><textarea name="message" maxlength="500" cols="19" rows="10"
+                            placeholder="Enter Text Here" required=""></textarea></label>
```

Comment:
- [ ] **OPTIONAL**
- kindly remove the `=""` from the `required attribute`. The required attribute is a boolean attribute that specifies that an input field must be filled out before submitting the form. It does not need any values added to it like other form attribute.

Reply:
ok thanks